groups/chips-uam-infrastructure/templates/chips_uam_user_data.tpl

```diff
@@ -13,10 +13,10 @@ yum -y install java-1.8.0-openjdk
 yum -y install Xvfb
 
 #UAM GUI
-mkdir -p /home/ec2-user/uam
-aws s3 cp s3://shared-services.eu-west-2.resources.ch.gov.uk/chips/uam/uam_gui-1.109.0-rc1.zip /home/ec2-user/uam
-unzip /home/ec2-user/uam/uam_gui-1.109.0-rc1.zip
-rm -rf /home/ec2-user/uam/uam_gui-1.109.0-rc1.zip
+cd /home/ec2-user/
```

Comment:
Check out pushd/popd for changing directories in shell:
https://unix.stackexchange.com/questions/77077/how-do-i-use-pushd-and-popd-commands

Reply:
pushd applied and tested.